readme.txt

```diff
@@ -0,0 +1,22 @@
+=== MoneyBadger Payments Gateway ===
```

Comment:
I need some way to build this into a plugin package, please add a script to do so

Reply:
ideally, having a docker-compose file that lets me easily spin up a dev environment would also be amazing